services/storage/src/simcore_service_storage/dsm.py

```diff
@@ -397,20 +378,23 @@ async def list_file(
 
         elif location == DATCORE_STR:
             # FIXME: review return inconsistencies
-            api_token, api_secret = self._get_datcore_tokens(user_id)
-            _dcw = DatcoreWrapper(api_token, api_secret, self.loop, self.pool)
-            data = []  # await _dcw.list_file(file_uuid)
-            return data
+            # api_token, api_secret = self._get_datcore_tokens(user_id)
+            import warnings
```

Comment:
maybe move this import

Reply:
hmm no I prefer to keep it where it is used. it should anyway not be used now.